DrrImageComputation/Logic/vtkSlicerDrrImageComputationLogic.cxx

```diff
@@ -1655,10 +1655,9 @@ vtkMRMLLinearTransformNode* vtkSlicerDrrImageComputationLogic::UpdateImageTransf
   }
 
   vtkNew<vtkSlicerIECTransformLogic> iecLogic;
-  iecLogic->SetMRMLScene(scene);
 
   // Update transforms in IEC logic from beam node parameters
-  iecLogic->UpdateIECTransformsFromBeam(beamNode);
+  BeamsLogic->UpdateIECTransformsFromBeam(beamNode);
```

Comment:
What is this `BeamsLogic` variable? Is it local? Then start with small letter. Is it member? Then need `this->` before. Same with `REVLogic` below